app/config/default_config.yml

```diff
@@ -1,6 +1,7 @@
 ---
 port: 8686
-data: "./data"
+data: "./local-data/scalabel"
```

Comment:
Should we provide a way to port existing projects from the old data dir? Otherwise this will break everyone's existing projects

Reply:
The data structure within the folder is not changed yet. Folder copying should work now.